### PR TITLE
[Documentation: README] : Punctuation

### PR DESCRIPTION
**Pull request by MayankMani00**
Added punctuation where required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ The Submitty project is hosted on [GitHub](https://github.com/Submitty).
 # Key Features
 
 * Secure testing of many languages: Python, C/C++, Java, Scheme, Prolog, SPIM, and anything available on GNU / Linux!
-* Customizable automated grading with immediate feedback to students
+* Customizable automated grading with immediate feedback to students.
```
Comment (shwehtom89) [inline README.md]: Periods are not necessary in this case as this is not a complete sentence. I believe part of the confusion comes from ! mark in the first item and etc. being an abbreviation for etcetera. Listed items do not necessarily need periods.

File: README.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ The Submitty project is hosted on [GitHub](https://github.com/Submitty).
...
 * Advanced grading tools: static analysis, JUnit, code coverage, memory debuggers, etc.
-* Student upload by drag-and-drop, zip upload, or version control
+* Student upload by drag-and-drop, zip upload, or version control.
```
Comment (shwehtom89) [inline README.md]: Same as above.

File: README.md [hunk 3]
```diff
@@ -21,9 +21,9 @@ The Submitty project is hosted on [GitHub](https://github.com/Submitty).
...
 * Correct mistakes through multiple submissions, flexible ``late day’’ policy, hidden tests
```
Comment (shwehtom89) [inline README.md]: These quotes do not match. Should be double quotes.

Comment (MasterOdin) [reply README.md]: Fixed in master, separate from this PR.

**Pull request closed without merging**